### ISSUE TITLE
GPU Usage 100% at dataloading while not proceeding

### ISSUE THREAD

**Issue by TintinWuNTUEE** [hustvl/4DGaussians#139]
Hi, I was trying to run the model via Colab provided at the README, however I encountered two issues, 
1. I was training on the bouncingball scene from dnerf and the GPU usage spiked to 100% while dataloading
2. I proceeded on training on the hellwarrior dataset and getting a loss is nan while training
(I am using a RTX 4090 with CUDA 11.7)

**Comment by guanjunwu** [comment]
e, most cases happens when your CUDA version not corresponds with your pytorch version.
Is your pytorch version = torch1.13+cu117?

**Comment by TintinWuNTUEE** [comment]
I am using torch1.13+cu116, does that make a difference?
